WireDomain/Sources/WireDomain/Event Decryption/UpdateEventDecryptor.swift

```diff
@@ -93,6 +110,24 @@ struct UpdateEventDecryptor: UpdateEventDecryptorProtocol {
                         attributes: logAttributes
                     )
                 }
+                
+            case .conversation(.mlsMessageAdd(let eventData)):
+                
+                WireLogger.updateEvent.info(
+                    "decrypting MLS event...",
+                    attributes: logAttributes
+                )
+
+                do {
+                    let decryptedEventData = try await mlsMessageDecryptor.decryptedEventData(from: eventData)
```

Comment:
similar to what we did for the proteusMessageAdd, we use the decryptor to decrypt the MLS messages from the event and return the same event with the decrypted messages (current one and buffered ones)